developer-docs-site/docs/nodes/validator-node/operator/running-validator-node/using-azure.md

```diff
@@ -5,16 +5,21 @@ slug: "run-validator-node-using-azure"
 
 # On Azure
 
-This is a step-by-step guide to install an Aptos node on Azure. Follow these steps to configure a validator node and a validator fullnode on separate machines. 
+This is a step-by-step guide to install an Aptos node on Microsoft Azure. Follow these steps to configure a validator node and a validator fullnode on separate machines. 
 
-:::danger Did you set up your Azure account?
+:::caution Did you set up your Azure account?
 This guide assumes that you already have Azure account setup.
 :::
 
+:::danger Do you have stale volumes after bumping your deployment's era?
```

Comment:
This is also redundant now that I have added an account to the Before list for consistency with the other cloud setup docs.